fudy/HomeScreen.qml

```diff
@@ -41,6 +45,19 @@ Item {
 				font.pointSize: 40
 				text: name
 			}
+
+			MouseArea {
+				anchors.fill: parent
+				onClicked: {
+					switch(model.name)
+					{
+					case "NOTE" : openApplication("qrc:/NoteScreen.qml"); break
+					case "PLAN" : openApplication("qrc:/PlanScreen.qml"); break
+					case "WORKSPACE" : openApplication("qrc:/WorkSpace.qml"); break
+					case "TODO" : openApplication("qrc:/TodoScreen.qml")
+					}
+				}
```

Comment:
```suggestion
				onClicked: {
					switch(model.name)
					{
					      case "NOTE" : openApplication("qrc:/NoteScreen.qml"); break;
					      case "PLAN" : openApplication("qrc:/PlanScreen.qml"); break;
					      case "WORKSPACE" : openApplication("qrc:/WorkSpace.qml"); break;
					      case "TODO" : openApplication("qrc:/TodoScreen.qml");break;
					}
				}
```
`  case "TODO" : openApplication("qrc:/TodoScreen.qml");break;` I add this `break` because when we add new modelName, other dev will add this break => change your line history in the git

Reply:
Do you think "NOTE", "PLAN", "WORKSPACE", "TODO" is good way?

Reply:
Add the default: